server/service/workspaceService.js

```diff
@@ -341,6 +367,65 @@ exports.saveWorkspace = async ( workspace ) => {
     }
 };
 
+
+/**
+ * Will save or update topics associated with a workspace.  
+ * Topics are passed as an Array of integers.  This function will replace any existing topics for
+ * the workspace with the topics represented by the topic id's passed.
+ * @param {Integer} workspaceId id of the topic 
+ * @param {*} topicIds Array of topic id's to be associated with the woorkspace
+ * @returns true for success / false for failure
+ */                                               
+exports.saveTopicsForWorkspace = async function( workspaceId, topicIds, topicsRequired ) {
+    // get the most recent version of the workspace
+    let text = "SELECT * from goals where id = $1";
+    let values = [ workspaceId ];
+    try {
+         
+        let res = await db.query( text, values );
+        
+        if( res.rowCount > 0 ) {
+
+            // first remove current resources associated with the topic
```

Comment:
Looks like this is left over from a previous file.
```suggestion
            // first remove current topics associated with the workspace
```